### PR TITLE
feat(apidocs): SCIM Team APIs

### PR DESCRIPTION
**Pull request by JoshFerge**
- API docs for the Teams portion of the SCIM API.
Screen Shot 2021-07-06 at 2 55 26 PM
Screen Shot 2021-07-06 at 3 02 31 PM
Screen Shot 2021-07-06 at 3 02 37 PM
Screen Shot 2021-07-06 at 3 02 48 PM
Screen Shot 2021-07-06 at 3 03 13 PM
Screen Shot 2021-07-06 at 3 06 52 PM
Screen Shot 2021-07-06 at 3 06 55 PM

depends on #27151 and #27154

### CODE REVIEW

**Comment by JoshFerge** [comment]
(fyi, references to the openapi.json file added in #27156)

File: api-docs/paths/scim/team_details.json [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+                ]
+            }
+        ]
+    },
+    "patch": {
+        "tags": [
+            "SCIM"
+        ],
+        "description": "Update a team's attributes with a SCIM Group PATCH Request. Valid Operations are:\n* Renaming a team: \n```json\n{\n    \"op\": \"replace\",\n    \"value\": {\n        \"id\": 23,\n        \"displayName\": \"newName\"\n    }\n}\n```\n* Adding a member to a team: \n```json\n{\n    \"op\": \"add\",\n    \"path\": \"members\",\n    \"value\": [\n        {\n            \"value\": 23,\n            \"display\": \"testexample@example.com\"\n        }\n    ]\n}\n```\n* Removing a member from a team: \n```json\n{\n    \"op\": \"remove\",\n    \"path\": \"members[value eq \\\"23\\\"]\"\n}\n``` \n*  Replacing an entire member set of a team: \n```json\n{\n    \"op\": \"replace\",\n    \"path\": \"members\",\n    \"value\": [\n        {\n            \"value\": 23,\n            \"display\": \"testexample2@sentry.io\"\n        },\n        {\n            \"value\": 24,\n            \"display\": \"testexample3@sentry.io\"\n        }\n    ]\n}\n```",
+        "operationId": "Update a Team's Attributes",
```
Comment (markstory) [inline api-docs/paths/scim/team_details.json:69]: Should `Team` be capitalized here? It isn't in the line above.

File: api-docs/paths/scim/team_index.json [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+                "$ref": "../../components/parameters/scim.json#/filter"
+            },
+            {
+                "$ref": "../../components/parameters/scim.json#/count"
+            },
+            {
+                "name": "excludedAttributes",
+                "in": "query",
+                "required": false,
+                "description": "fields that should be left off of return values. Right now the only supported field for this query is `members`.",
```
Comment (markstory) [inline api-docs/paths/scim/team_index.json:30]:
```suggestion
                "description": "Fields that should be left off of return values. Right now the only supported field for this query is `members`.",
```

File: api-docs/paths/scim/team_details.json (hunk 1 above)
Comment (JoshFerge) [reply api-docs/paths/scim/team_details.json:69]: "operationId" is in title format (what goes in the sidebar menu), so caps there.

**Review by markstory — APPROVED** [review]

**Pull request merged**